server/testsupport/timelineResponseTestDataBuilder.ts

```diff
@@ -0,0 +1,55 @@
+import type { TimelineResponse } from 'educationAndWorkPlanApiClient'
```

Comment:
This is the test data builder for the REST API type, and with the exception of my comment below is fine 👍 

Do we also have a test data builder for the view model type `Timeline` ?
We will need one, so you'll need to create a new test data builder file `timelineTestDataBuilder.ts` that builds a valid `Timeline` with a function `aValidTimeline`

Keep this test data builder because it's fine (after deleting `problemRetrievingData`); and add a new file `timelineTestDataBuilder.ts` that builds a valid `Timeline` with a function `aValidTimeline` 👍

Reply:
I've added a new `timelineTestDataBuilder.ts` data is pretty much the same as the response (for now) but with the additional `problemRetrievingData`